### DISCUSSION TITLE
Rozszerzenie strukutry ocen sprzedawcy o nowe pola

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#9978] [Suggestion - verification]
GET /sale/user-ratings
zwraca aktualnie:
listę ocen postaci:

`{'buyer': {'id': '44054668', 'login': 'NS'},
              'comment': '',
              'createdAt': '2024-09-26T12:05:15.404Z',
              'excludedFromAverageRates': False,
              'id': '66f54dfb3c7c4e034cc7c0b4',
              'lastChangedAt': '2024-09-26T12:05:15.404Z',
              'order': {'id': 'b249c7f0-7a8a-11ef-9e48-2d81ff6ba349',
                        'offers': [{'id': '7775816856',
                                    'title': 'E2E Świat Lodu i Ognia Elio M. '
                                             'García. Jr., George R. R. '
                                             'Martin'}]},
              'rates': {'description': 5, 'service': 5},
              'recommended': True,
              'removal': {'possibleTo': '2024-12-25T13:05:15.404Z'}},`
              
 Czy można rozszerzyć zwracane informacje o zaznaczone opcje przy wystawianiu oceny?:
![obraz](https://github.com/user-attachments/assets/2a513b75-38fc-43e8-9935-f3d87b48a57e)

**Comment by dzialdowski** [comment]
Podbijam - w zestawieniu z PUT /sale/user-ratings/{ratingId}/answer i przy pustej treści opinii bez tych danych o badgach ciężko będzie się domyśleć, o co chodziło Klientowi i jak mu odpowiedzieć. 
Pracownik, który chciałby odpowiedzieć Klientowi na opinię musi wtedy poza wewnętrzną aplikacją odwiedzić również stronę Allegro by mieć komplet danych o opinii.

**Reply by Lukasz-Zurek** [reply]
@dzialdowski Dzięki za sugestię! Masz rację - jest to jak najbardziej do aktualizacji po naszej stronie. Przekazałem już temat do właściwego zespołu. Jak tylko będziemy zapowiadać zmiany - dam znać w wątku.
